mlrun/runtimes/kubejob.py

```diff
@@ -56,19 +57,46 @@ def is_deployed(self):
             return True
         return False
 
-    def with_source_archive(self, source, pythonpath=None, pull_at_runtime=True):
+    def with_source_archive(
+        self, source, workdir=None, handler=None, pull_at_runtime=True
```

Comment:
Backwards incompatible change, why ?

Reply:
1. we want the same across runtimes, 2. i assume no one uses it, 3. it will still work since the pypath value can be used for the workdir if the user placed values by order. we can add last attr for pypath in case someone specified it explicitly but i think its an overkill